### PR TITLE
Feature/req 62 freigabeprozess

### PR DESCRIPTION
**Pull request by viktorianer**
For the ensuring high standards of quality it should be possible for the task creators to check the inputs of completed task and to decide subsequently, whether the changes will be accepted or rejected. Thereby, it should be possible to see the old and new provided Information in order to make a decision, which information will be taken over.  An activation of these release processes should be configurable.

### CODE REVIEW

File: sernet.gs.ui.rcp.main/src/sernet/verinice/bpm/rcp/TaskView.java [hunk 1]
```diff
@@ -804,6 +816,25 @@ public void removeTask(ITask task) {
         contentProvider.removeTask(task);
     }
 
+    public void closeEditorForElement(String uuid) {
+        for (IEditorReference editorReference : PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage().getEditorReferences()) {
+            try {
+                if (editorReference.getEditorInput() instanceof BSIElementEditorInput && uuid.equals(((BSIElementEditorInput) editorReference.getEditorInput()).getCnAElement().getUuid())) {
```
Comment (pixeldrama) [inline sernet.gs.ui.rcp.main/src/sernet/verinice/bpm/rcp/TaskView.java]: Train wreck.

**Comment by viktorianer** [comment]
> It is neccessary to have write access in order to change Properties in the CHANGE REQUEST context. So it is possible for the assignee to change properties without following the task release workflow. The assignee can write its changes directly to the cnatreeeelement. Maybe it would be better if the assignee only gains read access to the ancestor object of the task and gains write access only in the CHANGE REQUEST context.

No, the assignee should have write access at any time - independent to context.

**Comment by viktorianer** [comment]
> NullPointer in Webfrontend. Occurs if you are logged in as a not privileged user (user-default-group or something)

fixed

File: sernet.gs.ui.rcp.main/src/sernet/verinice/bpm/rcp/TaskView.java (hunk 1 above)
Comment (viktorianer) [inline sernet.gs.ui.rcp.main/src/sernet/verinice/bpm/rcp/TaskView.java]: fixed

**Pull request merged**